tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -110,8 +126,13 @@ spec:
 
             for manifest_digest in $manifest_digests; do
               for tag in ${TAGS}; do
-                # Iterate over both rh-registry-repo and registry-access-repo
                 for registry_reference in ${rh_registry_repo} ${registry_access_repo}; do
+                  # Check if the image is already signed
+                  if is_signed "${registry_reference}" "${tag}" "${manifest_digest}"; then
```

Comment:
We need the same for source containers as well.

Reply:
Added, thanks!